ejb3/src/main/java/org/jboss/as/ejb3/timerservice/TimerHandleImpl.java

```diff
@@ -69,16 +69,6 @@
      * @throws IllegalArgumentException If either of the passed parameters is null
      */
     public TimerHandleImpl(final String id, final String timedObjectId, final TimerServiceImpl service) throws IllegalArgumentException {
```

Comment:
Can you also remove the IllegalArgumentException from method signature, and javadoc?